src/zeit/importer/k4import.py

```diff
@@ -101,6 +101,8 @@ def run_dir(input_dir, product_id_in):
     cnames = []
 
     k4_files = os.listdir(input_dir)
+    boxes = {}
+    unique_ids = {}
```

Comment:
Hmm da sind aber doch keine uniqueIds drin, sondern das ist "articles by uniqueId"... insofern die Variable lieber `articles` nennen? (Dann haben wir boxes und articles, und jede Datei geht entweder in den einen oder den anderen Eimer.)